index.Rmd

```diff
@@ -9,4 +9,24 @@ link-citations: yes
 description: "This is a learning resource for using Git and Visual Studio Team Services (VSTS) for managing Data Science Projects."
 ---
 
-# Preface {-}
\ No newline at end of file
+# Preface {-}
+
+## Overview
+
+This book aims act as a resource for analysts on how and why they should use version control.
+
+It is in the context of the Department for Education Corporate environment and focusses on using Git and Visual Studio Team Services (the departments existing code version control tool of choice). 
+
+R and R Studio are used as code base we are version controlling and IDE and the examples will be a mix of using R studio and the command line. 
+
+## Ways of Working
+
+On a high level this resource is advocating a new way of working for analytical projets which can be summarised as follows:
+
+1. **Plan your workload up front** - This will help you not only better understad the project but write less, better code to get the job done.
```

Comment:
"understad the project" should be "understand the project"

Reply:
I'll fix that cheers

Reply:
Done - Actually a few more typos that got picked up with a spell check